test/scripts/traffic-flow-tests.sh

```diff
@@ -8,7 +8,7 @@ export OCI_BIN="${KIND_EXPERIMENTAL_PROVIDER:-docker}"
 export TFT_TEST_IMAGE="ghcr.io/ovn-kubernetes/kubernetes-traffic-flow-tests:latest"
 export TRAFFIC_FLOW_TESTS_DIRNAME="kubernetes-traffic-flow-tests"
 export TRAFFIC_FLOW_TESTS_REPO="https://github.com/ovn-kubernetes/kubernetes-traffic-flow-tests.git"
-export TRAFFIC_FLOW_TESTS_COMMIT="3e092f2c448bdffcddd5d025b97df53145ccea62"
+export TRAFFIC_FLOW_TESTS_COMMIT="94a22d237257254869fa93041aa93cbb338ce8c9"
```

Comment:
perhaps nice to think of a minor tagging or releases for this repo if its there ? :)

Reply:
maybe...

- I like git-sha sums, because they encode the exact content that is used. Contrary to git-tags, that (theoretically) can be force-pushed.
- If this is about having a git-tag (e.g. releases), then the only benefit is that "v1.3" reads nicer. I am not saying that is nothing, but if you want to know what this really is, you will need to checkout the sources, which you can also do via the git-sha sum.
- if this is about having a git-branch (and move it), then that would be nice, as we can just update the branch without needing to update the sources here. But this is also a downside. We currently use intentionally an exact version in git.

I am not against that, but I also don't see the benefit...